website/docs/docs/building-a-dbt-project/exposures.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Exposures"
+id: "exposures"
+---
+
+<Changelog>
+
+* Exposures are new in `v0.18.1`
+
+</Changelog>
+
+## Related documentation
+* [Exposure properties](exposure-properties)
+* [`exposure:` selection method](node-selection/methods#the-exposure-method)
+
+## Getting Started
+
+Exposures make it possible to define and describe a downstream use of your dbt project, such as in a dashboard, application, or data science pipeline. By defining exposures, you can then:
+- run, test, and list resources that feed into your exposure
+- populate a dedicated page in the auto-generated [documentation](documentation) site with context relevant to data consumers
+
+### Declaring an exposure
+
+Exposures are defined in `.yml` files in your `models` directory (as defined by the [`source-paths` config](source-paths)), nested under an `exposures:` key.
+
+<File name='models/<filename>.yml'>
+
+```yaml
+exposures:
+  
+  - name: weekly_jaffle_metrics
+    type: dashboard
+    maturity: high
+    url: https://bi.tool/dashboards/1
+    description: >
+      Did someone say "exponential growth"?
+    
+    depends_on:
+      - ref('fct_orders')
+      - ref('dim_customers')
+      - source('gsheets', 'goals')
+      
+    owner:
+      name: Claire from Data
+      email: data@jaffleshop.com
+```
+
+</File>
+
+### Available Properties
+
+_Required:_
+- **name** (must be unique among exposures)
+- **type**: one of `dashboard`, `notebook`, `analysis`, `ml`, `application`
```

Comment:
I wonder if it would be helpful to say more about these types (especially since it's required)?